### PR TITLE
notify 1.0.4 (new formula)

### PR DESCRIPTION
**Pull request by fopina**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/notify.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  head "https://github.com/projectdiscovery/notify.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/notify"
+  end
+
+  test do
+    assert_match "Current Version: ", shell_output("#{bin}/notify --version 2>&1")
```
Comment (carlocab) [inline Formula/notify.rb:16]:
```suggestion
    assert_match "Current Version: #{version}", shell_output("#{bin}/notify --version 2>&1")
```
Does this work?

Comment (fopina) [reply Formula/notify.rb]: it does indeed, updated

**Comment by carlocab** [comment]
LGTM. Could you squash your commits together please?

**Comment by fopina** [comment]
@carlocab done. Sorry, I always assume project maintainers squash on merge unless explicitly requested/agreed otherwise 😄

**Review by carlocab — DISMISSED** [review]
No problem; thanks for your contribution!

**Review by carlocab — APPROVED** [review]

**Comment by carlocab** [comment]
> ⚠️ @p-linnane bottle publish [failed](https://github.com/Homebrew/homebrew-core/actions/runs/4566676877).

Autosquash doesn't work when there have been recent changes to workflow files 😅 I'll need to look at a fix for that. Probably #126975.

**Comment by carlocab** [comment]
Merged in b09ed658fd1b59285e80442746bca36bb2f7d078. Thanks for your contribution again!

For future PRs, please ✅ the "Allow edits by maintainers" on the GitHub UI to make them less difficult to merge.

**Pull request closed without merging**